### DISCUSSION TITLE
In Composer DataprocCreateClusterOperator not working

### DISCUSSION THREAD

**Discussion by viralupadhyaya** [apache/airflow#17587] [Q&A] [answered]
I am using composer version 1.14.4 and airflow version 1.10.14. I want to use DataprocCreateClusterOperator but when I add code 

`from` airflow.providers.google.cloud.operators.dataproc import DataprocCreateClusterOperator`

receiving following error.
Broken DAG: [path/to/file.py] cannot import name 'Duration'.

Searched online but it seems no one else is getting this error. Any solution?

**Comment by turbaszek** [comment, marked as answer]
What version of Google python client do you have installed?

**Reply by viralupadhyaya** [reply]
I have Python version 3.

**Reply by turbaszek** [reply]
The question is more about version of `google-cloud-dataproc` package which is required by the operators. Also which version of Google backport provider do you use?

**Reply by viralupadhyaya** [reply]
Following installed in PYPI packages. Not sure if anything else required, please let me know.
google-cloud-dataproc==2.2.0
apache-airflow-backport-providers-ssh==020.10.29
apache-airflow-backport-providers-sftp==2020.10.29
protobuf==3.14.0

**Reply by viralupadhyaya** [reply]
I added apache-airflow-backport-providers-google==2021.3.3 and the error has gone away. Thank you.
